models/issues/issue_project.go

```diff
@@ -90,16 +90,16 @@ func LoadIssuesFromBoardList(ctx context.Context, bs project_model.BoardList) (m
 	return issuesMap, nil
 }
 
-// ChangeProjectAssign changes the project associated with an issue, if newProjectID is 0, the issue is removed from the project
-func ChangeProjectAssign(ctx context.Context, issue *Issue, doer *user_model.User, newProjectID, newColumnID int64) error {
+// IssueAssignOrRemoveProject changes the project associated with an issue, if newProjectID is 0, the issue is removed from the project
+func IssueAssignOrRemoveProject(ctx context.Context, issue *Issue, doer *user_model.User, newProjectID, newColumnID int64) error {
```

Comment:
It's just derived from old code. To reduce the changes to make review easier, so I haven't merge them as one function. If you think it's better, I can do it in this PR. Otherwise, I can do it in another PR.

Reply:
f7f36ddbaaf0619ce0d47bbb2d75763d56ca4a0c